src/dnsmasq/option.c

```diff
@@ -5961,7 +5961,10 @@ void read_opts(int argc, char **argv, char *compile_opts)
       add_txt("servers.bind", NULL, TXT_STAT_SERVERS);
       /* Pi-hole modification */
       add_txt("privacylevel.pihole", NULL, TXT_PRIVACYLEVEL);
```

Comment:
```suggestion

```

Do we still need this line? It was added by https://github.com/pi-hole/FTL/pull/417 and I don't really see a need for this to be open to every client to the network.

Reply:
There is actually more code involved here, I will remove this in a dedicated RP